flexmeasures/ui/templates/crud/asset.html

```diff
@@ -66,9 +60,19 @@
                 </div>
                 {% endif %}
             </div>
-            <form action="/assets/{{ asset.id }}/status" method="get">
-                <button id="asset-status-button" class="btn btn-sm btn-responsive" type="submit">Asset status</button>
-            </form>
+            <div class="header-action-button">
+                <div class="">
+                    <form action="/assets/{{ asset.id }}/status" method="get">
+                        <button id="asset-status-button" class="btn btn-sm btn-responsive" type="submit">Asset status</button>
+                    </form>
+                </div>
+                <div class="">
+                    <form action="/assets/auditlog/{{ asset.id }}" method="get">
```

Comment:
When I click this, the URL "http://localhost:5000/assets/auditlog/32?" loads which displays a 404 Not Found.

It seems there is an API call but this button would supposedly go to a UI view?